Tests/NIOEmbeddedTests/EmbeddedChannelTest.swift

```diff
@@ -13,7 +13,60 @@
 //===----------------------------------------------------------------------===//
 
 import XCTest
-@testable import NIO
+import NIOCore
+@testable import NIOEmbedded
+
+class ChannelLifecycleHandler: ChannelInboundHandler {
```

Comment:
Is that used anywhere?

Reply:
Yup, in the tests within this file.

Reply:
Found it! Thanks! Where did this code live before? Is this now duplicated?

Reply:
Yup, it lived in NIO’s channel tests. I don’t think we’re likely to bother deduplicating it though, it’s fairly trivial.